.travis.yml

```diff
@@ -16,8 +16,11 @@ matrix:
     - node_js: "4"
       env: PERCY_ENABLE=0
 before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 0.27.5
+  - export PATH=$HOME/.yarn/bin:$PATH
```

Comment:
Travis includes Yarn in their containers, but the version you get seems to be unreliable. We were previously getting the latest version by default, but this morning we started getting 0.17 instead. This just explicitly installs a modern version that has support for workspaces.